### ISSUE TITLE
NPE in EvalHelper

### ISSUE THREAD

**Issue by rmuller** [trimou/trimou#117]
Using template:

`{{eval '_mappings.language' locale}}`

If locale cannot be resolved a NPE is thrown. See below

I expect the "missing value value".

```
>>>>>> java.lang.NullPointerException
       org.trimou.handlebars.EvalHelper.execute(EvalHelper.java:92)
       org.trimou.engine.segment.HelperExecutionHandler.execute(HelperExecutionHandler.java:149)
       org.trimou.engine.segment.ValueSegment.execute(ValueSegment.java:88)
       org.trimou.engine.segment.AbstractContainerSegment.execute(AbstractContainerSegment.java:56)
       org.trimou.engine.segment.RootSegment.execute(RootSegment.java:47)
       org.trimou.engine.parser.Template.render(Template.java:119)
```

**Comment by mkouba** [comment]
Good point. `null` params should be probably ingored. Would you mind to send a PR?

**Comment by mkouba** [comment]
Fixed by #118.
